packages/rancher-prom2teams/generated-changes/patch/templates/deployment.yaml.patch

```diff
@@ -0,0 +1,110 @@
+--- charts-original/templates/deployment.yaml
++++ charts/templates/deployment.yaml
+@@ -2,8 +2,8 @@
+ kind: Deployment
+ metadata:
+   name: {{ include "prom2teams.fullname" . }}
+-  labels:
+-{{ include "prom2teams.labels" . | indent 4 }}
++  namespace: {{ include "prom2teams.namespace" . }}
++  labels: {{ include "prom2teams.labels" . | nindent 4 }}
+ spec:
+   replicas: {{ .Values.replicaCount }}
+   selector:
+@@ -16,17 +16,16 @@
+         app.kubernetes.io/name: {{ include "prom2teams.name" . }}
+         app.kubernetes.io/instance: {{ .Release.Name }}
+     spec:
+-    {{- with .Values.imagePullSecrets }}
+-      imagePullSecrets:
+-        {{- toYaml . | nindent 8 }}
+-    {{- end }}
++      {{- with .Values.imagePullSecrets }}
++      imagePullSecrets: {{ toYaml . | nindent 8 }}
++      {{- end }}
+       volumes:
+         - name: config
+           configMap:
+-            name: prom2teams-config
++            name: {{ include "prom2teams.fullname" . }}
+       containers:
+         - name: {{ .Chart.Name }}
+-          image: "{{ .Values.image.repository }}:{{ default .Chart.AppVersion .Values.image.tag }}"
++          image: {{ include "system_default_registry" . }}{{ .Values.image.repository }}:{{ default .Chart.AppVersion .Values.image.tag }}
+           imagePullPolicy: {{ .Values.image.pullPolicy }}
+           ports:
+             - name: http
+@@ -36,44 +35,45 @@
+             - name: config
+               mountPath: /opt/prom2teams/helmconfig/
+           env:
+-          - name: APP_CONFIG_FILE
+-            value: {{ .Values.prom2teams.config | quote }}
+-          - name: PROM2TEAMS_PORT
+-            value: {{ .Values.prom2teams.port | quote }}
+-          - name: PROM2TEAMS_HOST
+-            value: {{ .Values.prom2teams.ip | quote }}
+-          - name: PROM2TEAMS_CONNECTOR
+-            value: {{ .Values.prom2teams.connector | quote }}
+-          - name: PROM2TEAMS_GROUP_ALERTS_BY
+-            value: {{ .Values.prom2teams.group_alerts_by | quote }}
+-          resources:
+-            {{- toYaml .Values.resources | nindent 12 }}
+-        {{- if .Values.securityContext.enabled }}
++            - name: APP_CONFIG_FILE
++              value: {{ .Values.prom2teams.config | quote }}
++            - name: PROM2TEAMS_PORT
++              value: {{ .Values.prom2teams.port | quote }}
++            - name: PROM2TEAMS_HOST
++              value: {{ .Values.prom2teams.ip | quote }}
++            - name: PROM2TEAMS_CONNECTOR
++              value: {{ .Values.prom2teams.connector | quote }}
++            - name: PROM2TEAMS_GROUP_ALERTS_BY
++              value: {{ .Values.prom2teams.group_alerts_by | quote }}
++          resources: {{ toYaml .Values.resources | nindent 12 }}
```

Comment:
If the reason for this change is just indentation, let's remove it in favor of keeping indentation in line with upstream. Patch file should only ideally show what we *had* to change.

Reply:
fixed.